core/src/test/resources/META-INF/services/io.questdb.griffin.FunctionFactory

```diff
@@ -128,9 +128,11 @@ io.questdb.griffin.engine.functions.math.MulDoubleFunctionFactory
 io.questdb.griffin.engine.functions.math.MulLongFunctionFactory
 io.questdb.griffin.engine.functions.math.MulIntFunctionFactory
 
+# 'abs'
 io.questdb.griffin.engine.functions.math.AbsIntFunctionFactory
 io.questdb.griffin.engine.functions.math.AbsShortFunctionFactory
 io.questdb.griffin.engine.functions.math.AbsLongFunctionFactory
+io.questdb.griffin.engine.functions.math.AbsDoubleFunctionFactory
```

Comment:
to be reachable from production environment this class has to be added to module-info.java as well

Reply:
I added it [here](https://github.com/questdb/questdb/pull/318/commits/29a76aa74d6fe1330da6bf0859ac79cb679cb720#diff-5f9d46cb1237695971ba96a98c75dc63R134)